### ISSUE TITLE
The Brave lion icon is now in a circular border

### ISSUE THREAD

**Issue by dlfsjdfjodifjsodifjsdo** [brave/browser-android-tabs#556]
it used to be just a picture of the brave logo, but now the logo is in a white circle. On stock android 8.0

**Comment by bradleyrichter** [comment]
@dlfsjdfjodifjsodifjsdo Would you mind sharing a screen shot? I have not seen this yet.

**Comment by dlfsjdfjodifjsodifjsdo** [comment]
https://ibb.co/iAu5Ln

this has only happened to brave

**Comment by yachtcaptain23** [comment]
Is this for a notification? I know that in previous Android versions, notifications were forced to be round with a white border.

**Comment by dlfsjdfjodifjsodifjsdo** [comment]
@yachtcaptain23 no, i updated brave on the play store and it went from a borderless icon to a bordered icon

**Comment by pindogu** [comment]
that is actually a new Android feature. Its called "adaptive icons". In the settings you can choose between a circle, square, "squircle" and some more. The idea is for all apps to adapt like that. Brave has the white around it so it can fit the chosen form. If you want to get the old icon back you can change it by using a launcher that allows this (i use Nova Launcher).

**Comment by pindogu** [comment]
https://github.com/brave/browser-android-tabs/issues/519

here is someone asking for it lmao

**Comment by dlfsjdfjodifjsodifjsdo** [comment]
@pindogu so all the apps i have which are borderless haven't been updated?

**Comment by pindogu** [comment]
@dlfsjdfjodifjsodifjsdo
either that or they aren't making use of the feature. Only 1 % of devices have Android Oreo so most apps probably aren't in a rush to add this.

**Comment by bluz71** [comment]
Just installed Brave on Android since sync has now landed (thanks Brave team). Now starting my full migration to Brave away from Chrome.

But, the non-adaptive Brave icon is not nice to look at on my Google Pixel device. It is the Brave Lion inside a white circle as seen in this picture:

https://ibb.co/iAu5Ln

It is the amount of white that is not nice, especially compared to the Chrome icon which is a neat circle.

Ideally I think the white and orange colors need to be flipped. That is what Twitter did, usually the Twitter bird is light blue, but on Android the Twitter icon is a white bird on a blue background. In the Android Adaptive Icon world I think it may look better to have a white lion on an orange background. But I am no graphic designer.

Anyway, I am very keen to see this improved. Thanks.

**Comment by bluz71** [comment]
Here is the Twitter Android adaptive icon (white bird on blue background):

![twitter](https://user-images.githubusercontent.com/11382509/52320171-ec677000-2a21-11e9-960e-5b2cf639d6da.png)

Compared with the usual Twitter logo which is just a blue bird.

Maybe the Brave Android team could do the same thing to enable [Android Adaptive Icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive).
